src/sparseml/pytorch/image_classification/utils/helpers.py

```diff
@@ -286,7 +287,7 @@ def create_model(
         else:
             model, arch_key = result
 
-        return model, arch_key
+        return model, arch_key, checkpoint_path
```

Comment:
where is the checkpoint path used in the main script?

Reply:
It's used while initializing the `trainer`, more specifically it is used to read the recipe from checkpoint in `_setup_checkpoint_manager(...)` method [here](https://github.com/neuralmagic/sparseml/blob/d1584610ab6fc28782eb668102ea81577fe6f6dd/src/sparseml/pytorch/image_classification/utils/trainer.py#L307)